sitemap.xml

```diff
@@ -66,7 +63,4 @@
   <url>
     <loc>https://docs.wildfly.org/38/WildFly_and_WildFly_Preview.html</loc>
   </url>
-  <url>
-    <loc>https://docs.wildfly.org/38/wildscribe/</loc>
```

Comment:
Instead of removal this should be replaced with https://docs.wildfly.org/38/feature-pack/doc/index.html